### PR TITLE
add doc for `start-task --start-time`

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/4485

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-create-task.md [hunk 1]
```diff
@@ -45,6 +57,12 @@ start-task [ -s "mysql-replica-01"] ./task.yaml
 + `remove-meta`:
     - 可选
     - 如果设置，则在启动指定任务时会移除该任务之前存在的 metadata
++ `start-time`:
+    - 可选
+    - 对于增量任务，可以通过该参数大致指定任务起始位点，无需在任务配置文件中指定 binlog 位置。当该任务存在 checkpoint 时，通过这种方式启动任务会自动开启 safemode 直到同步过 checkpoint，以避免重置任务到更早位置时遇到数据重复的报错
```
Comment (ran-huang) [inline dm/dm-create-task.md:62]:
```suggestion
    - 对于增量任务，可以通过该参数大致指定任务起始位点，无需在任务配置文件中指定 binlog 位置。当该任务存在 checkpoint 时，如果通过这种方式启动任务，DM 会自动开启 safe mode 直到同步过 checkpoint，以避免重置任务到更早位置时遇到数据重复的报错
```

**Review by ran-huang — COMMENTED** [review]
spec 里提到的这个部分，文档里要体现出来吗？
```
任务开始时将会有三个起始点：

配置文件中指定的位置
checkpoint 记录的位置
start-task --start-time 指定的时间对应的位置

三个起始点如何选择，应该有文档进行明确说明；命令执行时也应该有所提示，明确的告知用户。
```

**Comment by lance6716** [comment]
> spec 里提到的这个部分，文档里要体现出来吗？
> 
> ```
> 任务开始时将会有三个起始点：
> 
> 配置文件中指定的位置
> checkpoint 记录的位置
> start-task --start-time 指定的时间对应的位置
> 
> 三个起始点如何选择，应该有文档进行明确说明；命令执行时也应该有所提示，明确的告知用户。
> ```

updated, PTAL

**Review by ran-huang — APPROVED** [review]
LGTM

File: dm/dm-create-task.md [hunk 2]
```diff
@@ -45,6 +57,13 @@ start-task [ -s "mysql-replica-01"] ./task.yaml
 + `remove-meta`:
     - 可选
     - 如果设置，则在启动指定任务时会移除该任务之前存在的 metadata
++ `start-time`:
+    - 可选
+    - 对于增量任务，可以通过该参数大致指定任务起始位点，该参数比任务配置文件中的 binlog 位置优先级更高，也比下游 checkpoint 中的 binlog 位置优先级更高
+    - 当该任务存在 checkpoint 时，如果通过这种方式启动任务，DM 会自动开启 safe mode 直到同步过 checkpoint，以避免重置任务到更早位置时遇到数据重复的报错
+    - 格式为 `'2021-10-21 00:01:00'` 或 `2021-10-21T00:01:00`
+    - 指定了过早的时间时，会从最早的 binlog 开始同步
+    - 指定了过晚的时间时，会报错 `start-time {input-time} is too late, no binlog location matches it`
```
Comment (sunzhaoyang) [inline dm/dm-create-task.md:66]: @lance6716 我记得设计的时候说还有个限制，指定过一次时间后，就不能再次指定一个 later 的时间了

**Review by sunzhaoyang — APPROVED** [review]
rest LGTM

File: dm/dm-create-task.md (hunk 2 above)
Comment (lance6716) [reply dm/dm-create-task.md:66]: 啊，这一个选项的注意事项好长啊……

Comment (lance6716) [reply dm/dm-create-task.md:65]: https://github.com/pingcap/docs-cn/pull/8640/commits/4ca1f015a12950e3b42c7c584a5c894af1f623c3

File: dm/dm-create-task.md [hunk 3]
```diff
@@ -16,13 +16,25 @@ help start-task
...
-  -s, --source strings   MySQL Source ID
+      --config string        Path to config file.
+      --master-addr string   Master API server address, this parameter is required when interacting with the dm-master
+      --rpc-timeout string   RPC timeout, default is 10m. (default "10m")
+  -s, --source strings       MySQL Source ID.
+      --ssl-ca string        Path of file that contains list of trusted SSL CAs for connection.
+      --ssl-cert string      Path of file that contains X509 certificate in PEM format for connection.
+      --ssl-key string       Path of file that contains X509 key in PEM format for connection.
+  -V, --version              Prints version and exit.
+
```
Comment (ran-huang) [inline dm/dm-create-task.md:37]:
```suggestion
```

**Pull request merged**